lib/runner/cli/cli.js

```diff
@@ -336,11 +336,29 @@ class CliRunner {
     return path.resolve(CliRunner.CONFIG_FILE_JS);
   }
 
+  transpileTStoJS(tsFile, filename) {
+    const ts = require('typescript');
+    const fs = require('fs');
+    const tsFileContent = fs.readFileSync(tsFile, 'utf8');
+
+    // Define TypeScript compiler options
+    const compilerOptions = {
+      target: ts.ScriptTarget.ES5,
+      module: ts.ModuleKind.CommonJS
+    };
+    const transpiledCode = ts.transpileModule(tsFileContent, {compilerOptions});
+    const jsCode = transpiledCode.outputText;
+    var Module = module.constructor;
```

Comment:
Instead of this, we can also put a `const Module = require('module');` statement at the top of the function definition (where other requires are done).